.circleci/config.yml

```diff
@@ -0,0 +1,91 @@
+# Java Gradle CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-java/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      # set a fixed version for the image circleci/openjdk:8-jdk-node-browsers
+      - image: circleci/openjdk:8-jdk-node-browsers
+        environment:
+          MAX_HEAP_SIZE: 2048m
+          HEAP_NEWSIZE: 512m
+
+      - image: cassandra:3.11
+        environment:
+          MAX_HEAP_SIZE: 1300m
+          HEAP_NEWSIZE: 512m
+
+      - image: scalarlabs/scalar-ledger:1.3.1
+        auth:
+          username: $DOCKERHUB_USERNAME
+          password: $DOCKERHUB_PASSWORD
+        environment:
+          SCALAR_CASSANDRA_HOST: 127.0.0.1
+          MAX_HEAP_SIZE: 500m
+          HEAP_NEWSIZE: 512m
+
+    working_directory: ~/scalardl-node-client-sdk
+
+    steps:
+      - checkout
+      # Download and cache dependencies.
+      - restore_cache:
+          keys:
+            - dependency-cache-{{ checksum "package-lock.json" }}
+
+      - run:
+          name: Install cqlsh
+          # We need to edit DEFAULT_CQLVER defined in the cqlsh script
+          # to connect to the server without --cqlversion command line option
+          command: |
+            curl -O https://bootstrap.pypa.io/get-pip.py
+            sudo python get-pip.py
+            sudo pip install cqlsh
+            sudo sed -i "s/^DEFAULT_CQLVER = .*/DEFAULT_CQLVER = '3.4.4'/" /usr/local/bin/cqlsh
+
+      - run:
+          name: Wait for Cassandra
+          # dockerize command is included in circleci/openjdk:8-jdk image
+          command: dockerize -wait tcp://127.0.0.1:9042 -timeout 1m
+
+      - run:
+          name: Install schema on Cassandra
+          command: cqlsh --execute "$SCHEMA_CONTENT" 127.0.0.1 9042
```

Comment:
So, it works fine ? Do we still need to remove spaces to update ?

Reply:
Yes it works fine, Yes, because for example when we wants to update the scheme, when we need to change from 
```DROP KEYSPACE IF EXISTS scalar;
CREATE KEYSPACE IF NOT EXISTS scalar WITH REPLICATION = {'class': 'SimpleStrategy', 'replication_factor': 1 };
CREATE KEYSPACE IF NOT EXISTS coordinator WITH replication = {'class': 'SimpleStrategy','replication_factor': 1 };

CREATE TABLE IF NOT EXISTS scalar.asset (
    id text,
    age int,
    argument text,
    before_argument text,
    before_contract_id text,
    before_hash blob,
    before_input text,
    before_output text,
    before_prev_hash blob,
    before_signature blob,
    before_tx_committed_at bigint,
    before_tx_id text,
    before_tx_prepared_at bigint,
    before_tx_state int,
    before_tx_version int,
    contract_id text,
    hash blob,
    input text,
    output text,
    prev_hash blob,
    signature blob,
    tx_committed_at bigint,
    tx_id text,
    tx_prepared_at bigint,
    tx_state int,
    tx_version int,
    PRIMARY KEY (id, age)
) WITH compaction = { 'class' : 'LeveledCompactionStrategy' };
```
to this 
```
DROP KEYSPACE IF EXISTS scalar; CREATE KEYSPACE IF NOT EXISTS scalar WITH REPLICATION = {'class': 'SimpleStrategy', 'replication_factor': 1 }; CREATE KEYSPACE IF NOT EXISTS coordinator WITH replication = {'class': 'SimpleStrategy','replication_factor': 1 };CREATE TABLE IF NOT EXISTS scalar.asset ( id text, age int, argument text, before_argument text, before_contract_id text, before_hash blob, before_input text, before_output text, before_prev_hash blob, before_signature blob, before_tx_committed_at bigint, before_tx_id text, before_tx_prepared_at bigint, before_tx_state int, before_tx_version int, contract_id text, hash blob, input text, output text, prev_hash blob, signature blob, tx_committed_at bigint, tx_id text, tx_prepared_at bigint, tx_state int, tx_version int, PRIMARY KEY (id, age) ) WITH compaction = { 'class' : 'LeveledCompactionStrategy' };
```

Reply:
Hmm, OK, I thought I told I want to avoid the conversion if possible.
Using some secret key for encryption doesn't work ?

Reply:
I haven't tried the key encryption but i think it will work, but before that, we have prepared a documentation regarding the overhead of these two methods https://docs.google.com/document/d/10E2lNMsiSIHiPyABeWwuxmQLkf5GiNvHLmnSoV1cXqY/edit#heading=h.b743t3mkndig.